src/Microsoft.Health.Fhir.Core/Features/Operations/Export/GetExportRequestHandler.cs

```diff
@@ -48,9 +48,11 @@ public async Task<GetExportResponse> Handle(GetExportRequest request, Cancellati
             }
             else if (outcome.JobRecord.Status == OperationStatus.Failed || outcome.JobRecord.Status == OperationStatus.Canceled)
             {
+                string failureReason = outcome.JobRecord.FailureDetails != null ? outcome.JobRecord.FailureDetails.FailureReason : Resources.UnknownError;
+                HttpStatusCode failureStatusCode = outcome.JobRecord.FailureDetails != null ? outcome.JobRecord.FailureDetails.FailureStatusCode : HttpStatusCode.InternalServerError;
```

Comment:
Is it the desired behavior to return a 500 error?

Reply:
When a job fails, the expectation is that the `FailureDetails` data is populated with the appropriate status code and we return that to the client. In the unexpected case that `FailureDetails` is not populated, we are defaulting to 500